README.md

```diff
@@ -28,6 +28,15 @@ docker-compose up -d
 SPRING_PROFILES_ACTIVE=local ./gradlew bootRun
 ```
 
+### Running with AWS LocalStack for SNS (Simple Notification Service)
+
+- Install the AWS CLI with Homebrew: `brew install awscli`
+- Configure the CLI for localstack: `aws configure --profile localstack` and set `AWS Access Key ID = test`, `AWS Secret Access Key = test`, `Default region name = eu-west-1`, `Default output format = json`
```

Comment:
It's a minor thing, but we won't use regions outside the UK, so for consistency, it's probably better to use
```suggestion
- Configure the CLI for localstack: `aws configure --profile localstack` and set `AWS Access Key ID = test`, `AWS Secret Access Key = test`, `Default region name = eu-west-2`, `Default output format = json`
```

Reply:
sure thing